Day-I-Social-Card/social-card/src/components/FooterComponents/Footer.js

```diff
@@ -1,31 +1,35 @@
-import React from 'react'
-import "./Footer.css"
+import React from 'react';
+import "./Footer.css";
 
```

Comment:
I am sorry this is the way I code. I don't like the top to be so close it too hard to read for me. I will narrow it down to two but otherwise I prefer it this way. It harder to tell where 1 part of the code starts and stops. Without comments it can make it much harder to read for me.